test/integration/client-side-encryption/range_explicit_encryption.prose.test.ts

```diff
@@ -0,0 +1,542 @@
+import { EJSON } from 'bson';
```

Comment:
I took a quick look at our file structure and we do have an established convention for the naming of these tests (see `client_side_encryption.prose.corpus.test.ts`), so let's follow it here, too; I suggest `client_side_encryption.prose.range.test.ts` but we can say range_explicit or whatever feels right to you

Reply:
My preference would be to just break it into a folder and then use a descriptive name.

`tests/integration/client-side-encryption/prose/range_explicit_encryptipnprose.test.ts` (or even `range_explicit_encryption.test.ts`) feels more logical than `client_side_encryption.prose.range.test.ts`.  It's also easier to determine from looking at the filename, when using a CMD+P for example.

Reply:
I think we still want to keep `.test.ts` as the file ending if it has mocha tests in it (unlike the deadlock tests which export them)